modules/virt-additional-scc-for-kubevirt-controller.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * virt/virt-additional-security-privileges-controller-and-launcher.adoc
+
+[id="virt-additional-scc-for-kubevirt-controller_{context}"]
+= Additional {product-title} Security Context Constraints and Linux capabilties for the `kubevirt-controller` service account
+
+Security Context Constraints (SCCs) control permissions for Pods. These permissions include actions that a Pod, a collection of containers, can perform and what resources it can access. You can use SCCs to define a set of conditions that a Pod must run with in order to be accepted into the system.
+
+The `kubevirt-controller` is a cluster controller that creates the virt-launcher Pods for virtual machines in the cluster. These virt-launcher Pods are granted permissions by the `kubevirt-controller` service account.
+
+== Additional SCCs granted to the `kubevirt-controller` service account
+
+The `kubevirt-controller` service account is granted additional SCCs and Linux capabilities so that it can create virt-launcher Pods with the appropriate permissions. These extended permissions allow virtual machines to take advantage of {VirtProductName} features that are beyond the scope of typical Pods. 
+
+The `kubevirt-controller` service account is granted the following SCCs:
+
+* `scc.AllowHostDirVolumePlugin = true` +
+This allows virtual machines to use the hostPath volume plug-in.
+
+* `scc.AllowPrivilegedContainer = false` +
+This ensures the virt-launcher Pod is not run as a privileged container.
+
+* `scc.AllowedCapabilities = []corev1.Capability{"NET_ADMIN", "NET_RAW", "SYS_NICE"}` +
+This provides the following additional Linux capablities
+`NET_ADMIN`,
+`NET_RAW`, and
+`SYS_NICE`.
+
+== Viewing the SCC and RBAC definitions for the `kubevirt-controller`
+
+You can view the `SecurityContextConstraint` definition for the `kubevirt-controller` by using the `oc` tool:
+
```

Comment:
should there be a `+` on this line?

Reply:
No, we usually add a '+' in this context in procedures to preserve the numbering. Since this is outside of a list it's not needed (and renders as a literal '+' if included).